dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/events/EventValidationTests.java

```diff
@@ -84,8 +88,7 @@
                 "E1123" ),
             Arguments.arguments( ouIdWithoutAccess, eventProgramId, eventProgramStageId,
                 "E1029" ),
-            Arguments.arguments( OU_ID, trackerProgramId, null, "E1123" ),
-            Arguments.arguments( OU_ID, trackerProgramId, eventProgramStageId, "E1089" ) );
```

Comment:
Why did you remove this test case? Program stage should still match the program

Reply:
The test case was removed because the program is now ignored in the payload, so the error cannot be triggered anymore.

The other case that now is there is a little be different and it is validating that the program linked by the program stage is the same as the program linked by the enrollment